dusty/commands/setup.py

```diff
@@ -64,6 +65,19 @@ def _get_and_configure_nginx_includes_dir():
     _get_raw_input('\n'.join(textwrap.wrap('You have a custom nginx config setup. Could not find an nginx.conf file. Please read our docs to see what is needed for the nginx config.  Once you have figured it out, please use `dusty config` command to adjust your `nginx_includes_dir`', 80)))
     return ''
 
+def _get_boot2docker_vm_size():
+    memory_gigs = 1.0 * virtual_memory().total / 10**9
+    if memory_gigs >= 15:
+        vm_gigs = 6
+        host_gigs = 16
+    else:
+        vm_gigs = 4
+        host_gigs = 8
+    if _get_raw_input('Your system seems to have {} gigabytes of memory. We would like to allocate {} to your vm. Is that ok? (y/n) '.format(host_gigs, vm_gigs)).upper() == 'Y':
```

Comment:
Can we not use the variable `host_gigs` and just stick with one measure of `memory_gigs`?  Arbitrarily setting `host_gigs` seems silly